c_glib/README.md

```diff
@@ -42,9 +42,74 @@ gobject-introspection gem based bindings.
 
 ## Install
 
+You can use packages or build by yourself to install Arrow GLib. It's
+recommended that you use packages.
+
 ### Package
```

Comment:
This is very nice. Can you add a note that these are unofficial packages? If there are "official" binary artifacts from an ASF project, they must accompany a release -- it would be great to make official .deb/.rpm packages on behalf of the project after the 0.3 release. Thanks @kou!